### PR TITLE
metrics-server: update metrics server to 2.11.2

### PR DESCRIPTION
**Pull request by ipochi**
Signed-off-by: Imran Pochi <imran@kinvolk.io>

### CODE REVIEW

**Review by surajssd — APPROVED** [review]

File: assets/charts/components/metrics-server/Chart.yaml [hunk 1]
```diff
@@ -12,4 +12,4 @@ maintainers:
 name: metrics-server
 sources:
 - https://github.com/kubernetes-incubator/metrics-server
-version: 2.11.1
+version: 2.11.2
```
Comment (invidian) [inline assets/charts/components/metrics-server/Chart.yaml:15]: This chart still ships metrics-server v0.3.6, we could upgrade to v0.3.7.

**Review by invidian — APPROVED** [review]
LGTM

File: assets/charts/components/metrics-server/Chart.yaml (hunk 1 above)
Comment (ipochi) [reply assets/charts/components/metrics-server/Chart.yaml:15]: why not wait for the official release of the helm chart ?

Comment (invidian) [reply assets/charts/components/metrics-server/Chart.yaml:15]: I'm fine with waiting, I'm just saying there is new image available :)

**Comment by surajssd** [comment]
So we can merge this I guess.

**Pull request merged**